### ISSUE TITLE
What to call Supporters and Supportees

### ISSUE THREAD

**Issue by Alonski** [Alonski/CoinPanion#6]
Patr for example calls them: Users and Creators

**Comment by rhlsthrm** [comment]
Creators and Supporters? Or do we want to not have the connotation that Creators must "create" something?

**Comment by Alonski** [comment]
I don't think Creators is the best term because we want to be broader than that.
Everyone can be both users essentially.
I can have my own profile and you can subscribe to me.
I can also subscribe to Tim if I want to later on.

**Comment by Josephrp** [comment]
everyone is a coinpanion, the sender is a coinpanion, the reciever is a coinpanion, sometimes the reciever will become a sender and yet, still, he will be a coinpanion. Everyone is a coinpanion :+1:  "coinpanion billion age" ; coinpanion 2018, coinpanion 2020. We're all coinpanions on the chain. at least in the front end it should be mentioned everywhere until the coinpanions see the coinpanion logo when the close their eyes.

**Comment by Alonski** [comment]
It could make sense that everyone has the same name: CoinPanion.
I am concerned that people will get confused when there isn't a clear cut between:
Supporters and Supportees

**Comment by rhlsthrm** [comment]
Let's come to a consensus on this and close the issue.

**Comment by Alonski** [comment]
Maybe we should just for the sake of simpleness call everyone Users and thats it?
Along with what I said about `Coined by user x` and `You coined user y`
I think this will be fine.
Thoughts?

**Comment by Josephrp** [comment]
I much prefer coinpanions (and to use it like "smurfs" in English or
"stroumpf" in French) but simplicity is more important than this... I do
like your idea re : "coined user y" :-) all is clear and fine. I don't
think this will be a very important success factor here.

**Comment by rhlsthrm** [comment]
Yes I agree with both points. Let's go with the "coined x.." "coined by x..." terminology for now. I think this issue can be closed.
